crypto/evp/evp_enc.c

```diff
@@ -675,3 +978,149 @@ int EVP_CIPHER_CTX_copy(EVP_CIPHER_CTX *out, const EVP_CIPHER_CTX *in)
         }
     return 1;
 }
+
+static void *evp_cipher_from_dispatch(int nid, const OSSL_DISPATCH *fns,
```

Comment:
why is this void*?

Reply:
'cause it's passed to a function that helps produce *any* type.